From 7ef6ff65e274c275c8e75d83aa67710d922af188 Mon Sep 17 00:00:00 2001
From: Igal Zeltser <izeltser@habana.ai>
Date: Sun, 10 Dec 2023 18:04:30 +0200
Subject: [PATCH] [SW-0] gaudi3: increase memcpy timeout

memcpy timeout is increased because during gaudi3 PO we saw that 1ms is
not enough for copying the PHY FW

Change-Id: I42c0e789d4d71122548b7812773543757dc20588
Signed-off-by: Igal Zeltser <izeltser@habana.ai>
---

diff --git a/arch/gaudi3/arc/common.c b/arch/gaudi3/arc/common.c
index 86936c2..403e42e 100644
--- a/arch/gaudi3/arc/common.c
+++ b/arch/gaudi3/arc/common.c
@@ -1190,7 +1190,8 @@
 #define MEM_CPY_SIZE_LIMIT		(SZ_1M - SZ_16)
 #define MEM_CPY_SIZE_SHIFT		4
 #define MEM_CPY_SIZE_SHIFT_MASK		(BIT(MEM_CPY_SIZE_SHIFT) - 1)
-#define HL_MEM_CPY_TIMEOUT_US		1000
+/* TODO: consider lowering timeout to 1ms, it was increased as a WA during PO */
+#define HL_MEM_CPY_TIMEOUT_US		10000
 
 int gaudi3_hbw_memcpy(uint64_t src, uint64_t dst, uint32_t size)
 {
